src/python_testing/TC_VALCC_4_3.py

```diff
@@ -66,12 +60,12 @@ async def test_TC_VALCC_4_3(self):
         self.step(1)
         attributes = Clusters.ValveConfigurationAndControl.Attributes
 
-        self.step(2)
+        self.step("2a")
         feature_map = await self.read_valcc_attribute_expect_success(endpoint=endpoint, attribute=attributes.FeatureMap)
 
         is_ts_feature_supported = feature_map & Clusters.ValveConfigurationAndControl.Bitmaps.Feature.kTimeSync
 
-        self.step(3)
+        self.step("2b")
         if not is_ts_feature_supported:
             logging.info("TimeSync feature not supported skipping test case")
```

Comment:
Can you swap for loop below to use this? https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/matter_testing_support.py#L1017. It'll make the TH happy and pretty.

Reply:
I will do this as a follow up PR.